cli/lib.rs

```diff
@@ -276,8 +277,8 @@ async fn info_command(flags: DenoFlags) {
   if let Err(e) = main_result {
     print_err_and_exit(e);
   }
-  print_file_info(worker.clone(), main_module.clone()).await;
-  let result = worker.await;
+  print_file_info(&worker, main_module.clone()).await;
+  let result = (&mut *worker).await;
```

Comment:
Is there some syntactic sugar that could be used here?

Reply:
*shrug* I'm not sure we should even have the worker be a future. I'd just leave it for now until we figure out the event loop.